pkg/spec/etcd_cluster_list.go

```diff
@@ -26,5 +24,5 @@ type EtcdClusterList struct {
 	// More info: http://releases.k8s.io/HEAD/docs/devel/api-conventions.md#metadata
 	unversioned.ListMeta `json:"metadata,omitempty"`
 	// Items is a list of third party objects
-	Items []spec.EtcdCluster `json:"items"`
+	Items []EtcdCluster `json:"items"`
```

Comment:
comment the public EtcdClusterList

// EtcdClusterList is a list of etcd clusters.